### ISSUE TITLE
Access to removable drives

### ISSUE THREAD

**Issue by 4tonyz** [puddletag/puddletag#585]
Puddletag installed on Ubuntu 20.04 only accesses folders on the hard drive and does not access folders on removable drives.
Unlike the file interface on Ubuntu these are not listed when opening a folder.

**Comment by sandrotosi** [comment]
yep, i can indeed replicate it, but i cannot pinpoint the location in our code where that window is built

**Comment by blueblots** [comment]
Is there something specific you're doing on the removable drive\puddletag to cause this, so far I haven't been able to replicate this on my end.

**Comment by animal6496** [comment]
I've a similar problem as reported by 4tonyz. I can access local folders but not those mapped on the server, they simply don't show in the file browser. I'm also on Ubuntu 20.04 and Nautilus/Nemo don't have problems browsing the remote music folders. Comming from Windows/Mp3tag I really like the functionality of puddletag  but the program is not very useful at the moment.

**Comment by sandrotosi** [comment]
I have not problem accessing directories via NFS or external drives.

Maybe you cant find them "easily" because they dont appear in the left column of "Open Folder" menu entry, like below (where it only shows my username and the / directory)?

![Screenshot from 2021-10-02 23-03-22](https://user-images.githubusercontent.com/346648/135737939-852bb309-77fe-4aab-8341-f43954458567.png)

**Comment by sabamimi** [comment]
I have just install it on Mint  21.2 (coming from windows MP3TAG) and I have the same issue regarding removable drives.
In Nemo file manager they appear under /media/xxxxx but /media is not listed in the Filesystem pane.
Am I missing something obvious?
Thanks for your feedback

**Comment by MartinEden** [comment]
I think the problem is that because puddletag is installed on Ubuntu as a snap, and snaps do not have access to removable drives by default.

For other apps I have fixed this in the past by running `sudo snap connect APP_NAME:removable-media`, but this does not work for puddletag, as the snap has to first register that it can use that interface.

So it might be as simple as updating the snap definition to allow it to be connected?
